### ISSUE TITLE
Refactoring awesome-vue structure?

### ISSUE THREAD

**Issue by mazipan** [vuejs/awesome-vue#2432]
It's very hard to maintain file `Readme.md` with very long contents.
Can we plan to refactoring this repo to be more easier to be added or updated in the future?

The easiest way to refactoring in my opinion is to split into each `.md` file for each categories in `awesome-vue`.
But for implement this things we need to freeze any PR maybe for few days.

Any comments or insight for this opinion?

**Comment by DannyFeliz** [comment]
And how are we going to show all the splited `.md` files as one?

**Comment by mazipan** [comment]
Just show the categories in `readme.md`, each categories will link to the `.md` file.

Or anybody have any other ideas?

**Comment by DannyFeliz** [comment]
@mazipan Yes but know this repo won't be "searchable" by default. As you can see basically all the awesome- repos are like this, the idea is having everything on one page where you can find quickly what you need.

**Comment by rmjordas** [comment]
I have a [repo](https://github.com/rmjordas/awesome-vue) that I try to keep up to date with changes in the official repo. It has some basic categorization split into multiple markdown files.

https://awesome-vue.js.org/

**Comment by Scrum** [comment]
@mazipan I don’t think it’s a good idea to search, given that you can’t always find wherever you are.
